qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -1,20 +1,19 @@
-- qemu_guest_agent: install setup image_copy unattended_install.cdrom
+- qemu_guest_agent: install setup image_copy unattended_install.cdrom single_driver_install.with_vioserial
     only Fedora, RHEL, Windows
     no Fedora.8, Fedora.9, Fedora.10, Fedora.11, Fedora.12, Fedora.13, Fedora.14, Fedora.15
     type = qemu_guest_agent
     gagent_name = "org.qemu.guest_agent.0"
     gagent_install_cmd = "rpm -q qemu-guest-agent || yum install -y qemu-guest-agent"
     Windows:
-        gagent_host_dep_install_cmd = "yum install -y mingw32-glib2"
-        gagent_host_install_cmd = "rpm -q qemu-guest-agent-win32 || rpm --ignorearch -ivh ftp://rpmfind.net/linux/centos/6.4/updates/x86_64/Packages/qemu-guest-agent-win32-0.12.1.2-2.355.el6_4_4.1.x86_64.rpm"
-        gagent_install_cmd = "net stop qemu-ga & C:\qemu-ga\qemu-ga.exe -s uninstall & C:\qemu-ga\qemu-ga.exe -s install"
-        gagent_dep_dlls = "/usr/i686-w64-mingw32/sys-root/mingw/bin/libglib-2.0-0.dll"
-        gagent_dep_dlls += " /usr/i686-w64-mingw32/sys-root/mingw/bin/iconv.dll"
-        gagent_dep_dlls += " /usr/i686-w64-mingw32/sys-root/mingw/bin/libintl-8.dll"
-        gagent_dep_dlls += " /usr/i686-w64-mingw32/sys-root/mingw/bin/libgcc_s_sjlj-1.dll"
-        gagent_host_path = "/usr/share/qemu-kvm/qemu-ga-win32/qemu-ga.exe"
+        i386:
+            qemu_ga_pkg = qemu-ga-x86.msi
+        x86_64:
+            qemu_ga_pkg = qemu-ga-x64.msi
+        gagent_download_cmd = "wget https://fedorapeople.org/groups/virt/virtio-win/direct-downloads/latest-qemu-ga/${qemu_ga_pkg}  -P /tmp/"
+        gagent_install_cmd = "C:\qemu-ga\${qemu_ga_pkg}  /quiet"
+        gagent_host_path = "/tmp/${qemu_ga_pkg}"
         gagent_guest_dir = "C:\qemu-ga"
-        gagent_remove_service_cmd = "net stop qemu-ga & del /f C:\qemu-ga\qemu-ga.exe & echo done"
+        gagent_remove_service_cmd = "net stop qemu-ga & del /f C:\qemu-ga\${qemu_ga_pkg} & echo done"
```

Comment:
If reinstall the qemu-ga-win with another version , it will uninstall the old one already be installed in the os. So we just need to confirm the installer is the one we want to test , and install it.